### ISSUE TITLE
Must have - copy and paste option in forum

### ISSUE THREAD

**Issue by ab668685** [topcoder-platform/beta-testing-forums#33]
copy and paste option in forum

![image](https://user-images.githubusercontent.com/61318135/109327667-97a02780-787e-11eb-90d5-a096ac2fd1c3.png)

**Comment by giosal** [comment]
I completely agree with this. It can be implemented in the same way as here, in GitHub. When you paste, it automatically uploads the image.
This is especially useful for Windows 10 and MacOS where when you take a screenshot (Win+Shift+S on Win10) it automatically copies the screenshot into clipboard.
And honestly, I don't really like when I have to save a screenshot just to upload it to somewhere. It's much better when it's copied to clipboard and you paste it without saving.
